### PR TITLE
Beta tags

### PR DESCRIPTION
**Pull request by keepingitneil**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -44,4 +44,4 @@ jobs:
           GOOGLE_APPLICATION_CREDENTIALS: google.json
         run: |
           echo $GOOGLE_CREDENTIALS_JSON > google.json
-          pytest --asyncio-mode=auto .
\ No newline at end of file
+          pytest --asyncio-mode=auto --timeout=60 .
```
Comment (theomonnom) [inline .github/workflows/tests.yml:47]: 💯

**Review by theomonnom — APPROVED** [review]
lgtm, sounds annoying to do lol

**Comment by theomonnom** [comment]
mb we can automate that in the future

**Comment by keepingitneil** [comment]
> mb we can automate that in the future

yeah agree

**Pull request merged**